auslib/web/common/rules.py

```diff
@@ -11,6 +12,8 @@
 
 
 def get_rules():
+    # We can't use a form here because it will enforce "csrf_token" needing
+    # to exist, which doesn't make sense for GET requests.
```

Comment:
I'm not sure I understand this comment. As far as I know, we don't use any WTForms stuff at all anymore.

Reply:
Ah, I brought this comment from the original function. 
Was removed.